setup.py

```diff
@@ -144,42 +152,66 @@ def run(self):
 # Setup arguments
 #==============================================================================
-setup_args = dict(name=NAME,
-      version=__version__,
-      description='Scientific PYthon Development EnviRonment',
-      long_description=
-"""Spyder is an interactive Python development environment providing
-MATLAB-like features in a simple and light-weighted software.
-It also provides ready-to-use pure-Python widgets to your PyQt5
-application: A source code editor with syntax highlighting and
-code introspection/analysis features, NumPy array editor, dictionary
-editor, Python console, etc.""",
-      download_url='%s/files/%s-%s.zip' % (__project_url__, NAME, __version__),
-      author="The Spyder Project Contributors",
-      url=__project_url__,
-      license='MIT',
-      keywords='PyQt5 editor shell console widgets IDE',
-      platforms=['any'],
-      packages=get_packages(),
-      package_data={LIBNAME: get_package_data(LIBNAME, EXTLIST),
-                    'spyder_breakpoints': get_package_data('spyder_breakpoints', EXTLIST),
-                    'spyder_profiler': get_package_data('spyder_profiler', EXTLIST),
-                    'spyder_pylint': get_package_data('spyder_pylint', EXTLIST),
-                    'spyder_io_dcm': get_package_data('spyder_io_dcm', EXTLIST),
-                    'spyder_io_hdf5': get_package_data('spyder_io_hdf5', EXTLIST),
-                    },
-      scripts=[osp.join('scripts', fname) for fname in SCRIPTS],
-      data_files=get_data_files(),
-      classifiers=['License :: OSI Approved :: MIT License',
-                   'Operating System :: MacOS',
-                   'Operating System :: Microsoft :: Windows',
-                   'Operating System :: POSIX :: Linux',
-                   'Programming Language :: Python :: 2.7',
-                   'Programming Language :: Python :: 3',
-                   'Development Status :: 5 - Production/Stable',
-                   'Topic :: Scientific/Engineering',
-                   'Topic :: Software Development :: Widget Sets'],
-      cmdclass=CMDCLASS)
+setup_args = dict(
+    name=NAME,
+    version=__version__,
+    description='The Scientific Python Development Environment',
+    long_description=(
+"""Spyder is a powerful scientific environment written in Python, for Python,
+and designed by and for scientists, engineers and data analysts.
+It features a unique combination of the advanced editing, analysis, debugging
+and profiling functionality of a comprehensive development tool with the data
+exploration, interactive execution, deep inspection and beautiful visualization
+capabilities of a scientific package.
+Furthermore, Spyder offers built-in integration with many popular
+scientific packages, including NumPy, SciPy, Pandas, IPython, QtConsole,
+Matplotlib, SymPy, and more.
+Beyond its many built-in features, Spyder's abilities can be extended even
+further via its plugin system and API.
```

Comment:
This

> Beyond its many built-in features, Spyder's abilities can be extended even
> further via its plugin system and API.

is not quite true in Spyder 3. It'll be in Spyder 4, but not right now, so please remove it

Reply:
Removed ``and API`` and changed to ``via third-party plugins.`` to imply the "system" isn't quite so developed at the moment. 

Do note that the verbatim phrase you cited currently appears in our website main page, Docs main page, Readme introduction, OpenCollective About section, (IIRC) that brochure I helped you with, various documents/proposals we've produced, etc. and others I don't remember at the moment since its part of our standard 4/5-line blurb that we use everywhere.